### ISSUE TITLE
Figure out how to distribute mongodb data for other contributors

### ISSUE THREAD

**Issue by HardcoreParkore** [HardcoreParkore/espn-ff-web#10]
I know nothing about this - this is basically my first time using mongodb not in a classroom environment.

In the short term, I assume I'll be exporting json and sending it to you somehow, but there's gotta be a better way to collaborate on this without having our local DBs hosed all the time.

**Comment by GabrielFabian** [comment]
You can host the DB on your server, similar to how you're hosting it locally. It'll take some configuration but it's the easiest way to ensure we are all working with the same data.

Thinking on this a bit more, we would likely want to set up a few DB's. Maybe `Prod`, `Test` and `Dev` on different ports so that we're not stepping on the production data.

**Comment by HardcoreParkore** [comment]
Ok that makes sense. I think it makes sense to do that after our initial deployment of the power rankings tool, just because we want to get it out asap. Until then I can send you guys payloads of what my DB looks like (it's hella simple right now).

Also, with a dev team this small, what's the boost we get from test vs dev? I assume we'd need to set up a test version of the web app on the server to integration test our deployments if we leverage that DB

**Comment by HardcoreParkore** [comment]
I like it. I think after first deployment I'll work to set those up.

**Comment by GabrielFabian** [comment]
Is the code for the actual server hosted somewhere else?

**Comment by HardcoreParkore** [comment]
nah it's in the /server directory

**Comment by HardcoreParkore** [comment]
No, the actual server is a digitalocean droplet that I've just been manually configuring, cowboy style. I'll work to back up the config, etc soon as well.

**Comment by HardcoreParkore** [comment]
#13
